site/src/util/mdxComponents.tsx

```diff
@@ -99,6 +108,21 @@ export const mdxComponents: Record<string, React.ReactNode> = {
       <a id={props.id} />
     );
   },
+  table: ({ children, ...props }: HTMLProps<HTMLTableElement>) => (
+    <Box
+      sx={{
+        "td, th": {
+          border: ({ palette }) => `1px solid ${palette.gray[30]}`,
+          paddingY: 1,
+          paddingX: 3,
+          typography: "bpSmallCopy",
+        },
+        marginBottom: 2,
+      }}
+    >
+      <table {...props}>{children}</table>
+    </Box>
```

Comment:
Is there a reason you're wrapping the `<table>` in a `<div>`? If you did this just to get the `sx` prop you could've also done:
```suggestion
    <Box
      component="table"
      sx={{
        "td, th": {
          border: ({ palette }) => `1px solid ${palette.gray[30]}`,
          paddingY: 1,
          paddingX: 3,
          typography: "bpSmallCopy",
        },
        marginBottom: 2,
      }}
      {...props}
    >
      {children}
    </Box>
```